app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PostsController < ApplicationController
+  def index
+    @posts = @user.posts
+  end
+
+  def show
+    # specific posts
+  end
```

Comment:
- [ ] [Optional] Kindly add code for rendering spacifc post to the show action. For example:

```
 @post = Post.find(request.params['id'])
```